Source/Public/Invoke-DockerTests.ps1

```diff
@@ -7,7 +7,7 @@ function Invoke-DockerTests {
 
         [Parameter(ValueFromPipelineByPropertyName = $true)]
         [String[]]
```

Comment:
I guess I didn't think to test the scenario with multiple paths. I will add that. thanks.

Reply:
Sorry, what i meant was, we should support only 1 config path. If the user wants to have multiple paths, then he should have them as sub folders under the config path. 😅

Reply:
I agree with your point about the recursive search. I am not sure I understand why it's bad to allow more than one path for the scenario where you tests resides in different folders next to each other instead of in a nested structure.

Perhaps we can simply support both? For the caller of the code, it does not matter, because they can still invoke the cmdlet with a [String] instead of [String[]]